src/constants/lists.ts

```diff
@@ -14,6 +14,7 @@ const WRAPPED_LIST = 'wrapped.tokensoft.eth'
 export const OPTIMISM_LIST = 'https://static.optimism.io/optimism.tokenlist.json'
 export const ARBITRUM_LIST = 'https://bridge.arbitrum.io/token-list-42161.json'
 export const CELO_LIST = 'https://celo-org.github.io/celo-token-list/celo.tokenlist.json'
+export const BOBA_LIST = 'https://raw.githubusercontent.com/bobanetwork/token-list/main/boba.tokenlist.json'
```

Comment:
we could also use this github pages url instead?
https://bobanetwork.github.io/token-list/boba.tokenlist.json

Reply:
URL has been updated